proxmox/resource_vm_qemu.go

```diff
@@ -456,8 +456,8 @@ func resourceVmQemu() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"ide0": schema_Ide("ide0"),
 									"ide1": schema_Ide("ide1"),
-									"ide2": schema_Ide("ide2"),
```

Comment:
Why the change from ide3 to ide2?

Reply:
am I right with this change since ide2 should be not used directly? Ora maybe it can be used but it conflicts with cloudinit is someone want to deploy a vm with a ISO file with terraform?

Reply:
Well `ide2` used to be reserved for the iso propertie, but that was removed in #937. Currently only `ide3` is reserved for `cloud-init` but we could move that to the new `disks` schema as well and let people chose on which device they'll mount the `cloud-init` disk. Fro now i would leave it as `ide3` as it's more intuitive that ide 0,1,2 are usable instead of 0,1,3.